pyscript.core/package.json

```diff
@@ -58,7 +58,7 @@
         "@ungap/with-resolvers": "^0.1.0",
         "@webreflection/idb-map": "^0.3.1",
         "basic-devtools": "^0.1.6",
-        "polyscript": "^0.15.6",
+        "polyscript": "^0.15.7",
```

Comment:
Something went wrong with the previous MR ... this should be `0.15.8` instead! That being said, it's not clear to me why this file shows up with changes already merged in main ... I think you should rebase properly then change only this version of polyscript (or I can file a new MR).

Reply:
Ack

Reply:
Hmm... this update clearly hasn't landed in main: https://github.com/pyscript/pyscript/blob/main/pyscript.core/package.json#L61 (I still see 0.15.7)

Reply:
yes, it doesn't mean it cannot land in here, it's the same code more or less, just one less PR to approve ;-)

Reply:
this is still outdated ... can you please make it `0.15.8` ?

Reply:
Done. Dammit - I knew I'd missed something,